source/access.tex

```diff
@@ -116,8 +116,9 @@
 the class appearing outside the class's \grammarterm{member-specification}{.}
 \begin{note} This access also applies to implicit references to constructors,
 conversion functions, and destructors. \end{note}
-\begin{example}
 
+\pnum
+\begin{example}
```

Comment:
There a star-like thing on the upper left corner of each diff hunk that, when pressed, shows more context.

Anyway, I did consider for each change whether I should create a new numbered paragraph for an example or not, and I decided on an individual basis. For this particular case, the example clearly does not belong to the preceding note, and I figured a separate paragraph would be more appropriate.